README.md

```diff
@@ -154,6 +155,56 @@ Result:
 26685 ms 99th, 
 28510 ms 99.9th.
 ```
+
+Command line example of KAFKA storage driver:
+```
+docker run --network host \
+emcmongoose/mongoose-storage-driver-kafka:4.2.8 \
+--load-batch-size=8196 \
```

Comment:
please try
load-batch-size=100 & storage-driver-limit-queue-input=10000
I think this will produce higher rates